src/DeviceInterface/initFormSubmissionsApi.js

```diff
@@ -59,6 +58,13 @@ export function initFormSubmissionsApi (forms) {
                     filledForm?.submitHandler('global pointerdown event + filled form')
                 }
             }
+
+            // TODO: Temporary hack to support Google signin in different languages
```

Comment:
Do we want to add an Asana task URL to these todo comments so we know where they're being tracked?